the-collab-lab/tcl-13-smart-shopping-list
# Rf rb item detail pretty

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Re-styled AddItem , ItemDetail , Nav bar and NewList components. Moved Header into Nav with conditional styling for nav items. Created new collection in database to store human-readable list names.

## Related Issue

closes #16 closes #17 

## Acceptance Criteria

The item detail screen (the screen a user sees when they tap on one of their items in the shopping list) looks something like the following (or equivalent, use your creativity!):

The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="700" alt="NewList - Before" src="https://user-images.githubusercontent.com/10679592/100176727-a9d88700-2e85-11eb-9c2f-e9d0bb3ea1cb.png">
<img width="1116" alt="AddItem - Before" src="https://user-images.githubusercontent.com/10679592/100176739-af35d180-2e85-11eb-94f7-b410ed4dd3bd.png">
<img width="1120" alt="ItemDetail - Before" src="https://user-images.githubusercontent.com/10679592/100176744-b0ff9500-2e85-11eb-9025-7dd135bb0fef.png">
<img width="228" alt="Nav-Before" src="https://user-images.githubusercontent.com/10679592/100176804-cd033680-2e85-11eb-969b-943c2b3682e6.png">


### After

<img width="703" alt="NewList - After" src="https://user-images.githubusercontent.com/10679592/100176754-b6f57600-2e85-11eb-8ab8-d12d73d916b4.png">
<img width="711" alt="AddItem - After" src="https://user-images.githubusercontent.com/10679592/100176787-c4aafb80-2e85-11eb-9621-09491787cd10.png">
<img width="700" alt="ItemDetail - After" src="https://user-images.githubusercontent.com/10679592/100176793-c83e8280-2e85-11eb-814f-f7cf444311cc.png">




## Testing Steps / QA Criteria

When creating a new list, check the database to confirm that the list name is being saved to listName collection
Navigate the app, look for the pretty.
